uw-frame-components/css/buckyless/general.less

```diff
@@ -36,6 +36,15 @@ button:focus {
   outline: none !important;
 }
 
+.aria-only {
```

Comment:
In general this CSS class is giving me pause. Admitting I'm no expert, I'd assumed all the necessary ARIA effects were achievable via ARIA attributes on elements without resorting to this sort of thing.